libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -137,19 +182,19 @@ class DiscoveryProvider {
       isCreator,
       minBlockNumber
     )
-    return this._makeRequest(req)
+    return await this._makeRequest(req)
```

Comment:
so eslint is saying that its preferable to await no matter what cause then if an error is thrown, it has this file in the stacktrace, otherwise it doesn't realize it's gone through here, if that makes sense?